### ISSUE TITLE
gapps.xml error

### ISSUE THREAD

**Issue by 46620** [phhusson/treble_manifest#13]
https://git.rip was taken offline and the repo fails to sync gapps:

```
fatal: unable to access 'https://git.rip/mtkgms/partner_gms/': server certificate verification failed. CAfile: none CRLfile: none
error: Cannot fetch mtkgms/partner_gms from https://git.rip/mtkgms/partner_gms
```

https://www.reddit.com/r/Piracy/comments/m44ae2/gitrip_has_been_seized_by_the_fbi/

**Comment by phhusson** [comment]
Yes that's an issue. At the moment it is being worked around with https://github.com/phhusson/treble_experimentations/blob/master/cloud/docker.sh#L86

7 avr. 2021 22:27:51 46620 ***@***.***>:

**Comment by 00p513-dev** [comment]
This has now been fixed

**Comment by 46620** [comment]
Fixed in #14 and merged
